package.json

```diff
@@ -27,6 +27,9 @@
     "pack": "electron-builder --dir",
     "dist": "electron-builder",
     "mas": "electron-builder --mac mas",
+    "setup-gitlint-hooks": "node ./scripts/gitlint/setup-gitlint-hook",
+    "lint-commits": "node ./scripts/gitlint --all-commits",
+    "lint-fix": "xo --fix",
```

Comment:
Can you remove `lint-fix` because `xo --fix` doesn't work all the time and may give you unexpected results.